styles.css

```diff
@@ -0,0 +1,224 @@
+*{
```

Comment:
formatting there should be a single space between the selector and the curly brace. Please correct this everywhere